geoips/plugins/modules/procflows/config_based.py

```diff
@@ -1985,7 +2025,13 @@ def call(fnames, command_line_args=None):
                             else:
                                 rgb_var = None
                             alg_xarray = combine_preproc_xarrays_with_alg_xarray(
-                                pre_proc, alg_xarray, rgb_var=rgb_var
+                                pre_proc,
+                                alg_xarray,
+                                rgb_var=rgb_var,
+                                covg_plugin=covg_plugin,
+                                covg_varname=covg_varname,
```

Comment:
This is ONLY passing the covg informative in sowe can print out the total coverage with each additional swath, should I comment that in my 1.17.2 patch?

Also we do not currently have a test in the geoips repo for this functionality